src/main/scala/io/iohk/ethereum/blockchain/sync/BlacklistSupport.scala

```diff
@@ -28,9 +30,12 @@ trait BlacklistSupport {
 
   def isBlacklisted(peerId: PeerId): Boolean =
     blacklistedPeers.exists(_._1 == peerId)
+
+  def handleBlacklistMessages: Receive = {
+    case UnblacklistPeer(ref) => undoBlacklist(ref)
+  }
 }
 
 object BlacklistSupport {
-  case class BlacklistPeer(peerId: PeerId, reason: String)
```

Comment:
we no longer need this message as all blacklisting happen inside the actor, we can just call a method